.teamcity/src/main/kotlin/model/CIBuildModel.kt

```diff
@@ -286,7 +286,9 @@ enum class TestType(val unitTests: Boolean = true, val functionalTests: Boolean
     // run integMultiVersionTest with all version to cover
     allVersionsIntegMultiVersion(false, true, false),
     parallel(false, true, false),
-    noDaemon(false, true, false, 300),
+    // FIXME: timeout increased because of https://github.com/gradle/gradle-private/issues/3695
+    //        revert it to 300 after fixed
+    noDaemon(false, true, false, 360),
```

Comment:
Let's hope 6 hours is enough.

Reply:
I would be conservative and nudge it hour-by-hour.